python-checks/src/test/resources/checks/genericFunctionTypeParameter.py

```diff
@@ -0,0 +1,23 @@
+def non_compliant(xx):
```

Comment:
I feel we could have some more tests. For example, having `TypeVar` imported from somewhere else than `typing` or imported as a different name.
Maybe stuff where the `TypeVar` has multiple assignments.
Basically the usual stuff, but I feel it would document how the rule is implemented and should behave.